src/engine/EntityComponentSystem/Components/TransformComponent.ts

```diff
@@ -18,15 +18,16 @@ export enum CoordPlane {
 }
 
 export class TransformComponent extends Component<'transform'> {
+  static type: 'transform';
```

Comment:
Do we need both static and instance property?

Reply:
No, I think this was leftover from an experiment